plugins/woocommerce/tests/api-core-tests/global-setup.js

```diff
@@ -1,8 +1,200 @@
-const { site } = require( './utils' );
+const { chromium, expect } = require( '@playwright/test' );
+const { GITHUB_TOKEN, UPDATE_WC } = process.env;
+const { downloadZip, deleteZip } = require( './utils/plugin-utils' );
+const axios = require( 'axios' ).default;
 
 module.exports = async ( config ) => {
-	// If BASE_URL is configured, we can assume we're on CI
-	if ( process.env.API_BASE_URL ) {
-		await site.reset( process.env.USER_KEY, process.env.USER_SECRET );
+	const { baseURL, userAgent } = config.projects[ 0 ].use;
+	const contextOptions = { baseURL, userAgent };
+
+	const browser = await chromium.launch();
+	const setupContext = await browser.newContext( contextOptions );
+	const setupPage = await setupContext.newPage();
+
+	// If API_BASE_URL is configured and doesn't include localhost, running on daily host
+	if (
+		process.env.API_BASE_URL &&
+		! process.env.API_BASE_URL.includes( 'localhost' )
```

Comment:
Tightened up this check, so you should be able to use a .env for localhost now and not trigger the site reset stuff.